.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: todoapp-daemon
+  namespace: todoapp
+  labels:
+    app: todoapp-daemon
+spec:
+  selector:
+    matchLabels:
+      app: todoapp-daemon
+  template:
+    metadata:
+      labels:
+        app: todoapp-daemon
+    spec:
+      containers:
+        - name: todoapp-daemon
+          image: curlimages/curl:latest
+          command:
+            ["sh", "-c", "watch -n 5 curl http://todoapp.todoapp.svc.cluster.local"]
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "100Mi"
+            limits:
+              cpu: "200m"
+              memory: "200Mi"
+          ports:
+          - containerPort: 80
+            hostPort: 30008
```

Comment:
![image](https://github.com/user-attachments/assets/6baa93c1-1b8c-4b62-89cb-33a0392c79a5)
Add Eof(End-of-file), need to add a blank line at the end

Reply:
fixed in commit "fix EoF in daemonset.yml"